FAQ.md

```diff
@@ -96,3 +96,15 @@ While this can be helpful for short-term fixes, it is important to remember that
 is in place, as the cluster currently has [no mechanism for reporting](https://github.com/openshift/machine-config-operator/issues/945) that the node has been changed in this
 fashion.  This kind of package replacement can also leave your nodes exposed to potential problems
 that are fixed in newer versions of the package.
+
+## Q: How do I build my own version of RHCOS for testing?
+
+You need the RHCOS manifest configuration (currently hosted on an RHT internal GitLab repo) and
```

Comment:
Maybe put an url.corp.redhat.com url to point to it?

Reply:
Can do!